src/foam/comics/v2/DAOControllerConfig.js

```diff
@@ -61,6 +61,20 @@ foam.CLASS({
       name: 'browseSubtitle',
       expression: function(of) { return "View all " + of.model_.plural.toLowerCase() +'.' }
     },
+    {
+      class: 'foam.u2.ViewSpecWithJava',
```

Comment:
Why is this needed? Can't we just refine rather than extend ViewSpec and then just use the regular ViewSpec?